src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope_NES.c

```diff
@@ -502,8 +516,964 @@ s32 D_8082BEB8[] = { 0, 0 };
+
+            pauseCtx->iconItemSegment =
+                (void*)(((uintptr_t)play->objectCtx.spaceStart + 0x30) & ~0x3F); // Messed up ALIGN64
+            size0 = SEGMENT_ROM_SIZE(icon_item_static_old);
+            func_80178E7C(SEGMENT_ROM_START(icon_item_static_test), pauseCtx->iconItemSegment, size0);
+
+            pauseCtx->iconItem24Segment = (void*)ALIGN16((uintptr_t)pauseCtx->iconItemSegment + size0);
+            size1 = SEGMENT_ROM_SIZE(icon_item_24_static_old);
+            func_80178E7C(SEGMENT_ROM_START(icon_item_24_static_test), pauseCtx->iconItem24Segment, size1);
+
+            pauseCtx->iconItemAltSegment = (void*)ALIGN16((uintptr_t)pauseCtx->iconItem24Segment + size1);
+            size2 = SEGMENT_ROM_SIZE(icon_item_gameover_static);
+            DmaMgr_SendRequest0(pauseCtx->iconItemAltSegment, SEGMENT_ROM_START(icon_item_gameover_static), size2);
+
+            pauseCtx->iconItemLangSegment = (void*)ALIGN16((uintptr_t)pauseCtx->iconItemAltSegment + size2);
+            DmaMgr_SendRequest0(pauseCtx->iconItemLangSegment, SEGMENT_ROM_START(icon_item_jpn_static),
+                                SEGMENT_ROM_SIZE(icon_item_jpn_static));
+
+            gSaveContext.timerStates[TIMER_ID_MOON_CRASH] = TIMER_STATE_OFF;
+
+            sGameOverPrimR = 255;
+            sGameOverPrimG = 130;
+            sGameOverPrimB = 0;
+            sGameOverPrimAlpha = 0;
+            sGameOverEnvR = 30;
+            sGameOverEnvG = 0;
+            sGameOverEnvB = 0;
+            sGameOverRectPosY = 98;
+            sGameOverColorTimer = 30;
+
+            pauseCtx->promptChoice = PAUSE_PROMPT_YES;
+            pauseCtx->state++;
+            if (gameOverCtx->state == GAMEOVER_INACTIVE) {
+                pauseCtx->state++;
+            }
+            break;
+
+        case PAUSE_STATE_GAMEOVER_3:
+            stepR = ABS_ALT(sGameOverPrimR - 30) / sGameOverColorTimer;
+            stepG = ABS_ALT(sGameOverPrimG - 0) / sGameOverColorTimer;
+            stepB = ABS_ALT(sGameOverPrimB - 0) / sGameOverColorTimer;
+            stepA = ABS_ALT(sGameOverPrimAlpha - 255) / sGameOverColorTimer;
+            if (sGameOverPrimR >= 30) {
+                sGameOverPrimR -= stepR;
+            } else {
+                sGameOverPrimR += stepR;
+            }
+            if (sGameOverPrimG >= 0) {
+                sGameOverPrimG -= stepG;
+            } else {
+                sGameOverPrimG += stepG;
+            }
+            if (sGameOverPrimB >= 0) {
+                sGameOverPrimB -= stepB;
+            } else {
+                sGameOverPrimB += stepB;
+            }
+            if (sGameOverPrimAlpha >= 255) {
+                sGameOverPrimAlpha -= stepA;
+            } else {
+                sGameOverPrimAlpha += stepA;
+            }
+
+            stepR = ABS_ALT(sGameOverEnvR - 255) / sGameOverColorTimer;
+            stepG = ABS_ALT(sGameOverEnvG - 130) / sGameOverColorTimer;
+            stepB = ABS_ALT(sGameOverEnvB) / sGameOverColorTimer;
```

Comment:
```suggestion
            stepB = ABS_ALT(sGameOverEnvB - 0) / sGameOverColorTimer;
```
To mirror the ones above.

Reply:
A few of these can probably have defines too (I thought they'd be regs in OoT, but they're identical to here), but that can come later.